quantum/frequently_asked_quantum_questions.rst

```diff
@@ -0,0 +1,55 @@
+
+*****************************
+Frequently Asked Questions
+*****************************
+
+How do quantum computers differ from classical computers? 
+=========================================================
+
+Conventional/classical computing utilizes information storage based on digital 
+devices storing “bits”, which are in either of two distinct states at a given time, 
+i.e. 0 or 1. Quantum computers utilize properties of quantum mechanics, such as 
+superposition and entanglement, in order to exceed certain capabilities of classical 
+computers. Superposition means that the units of information storage can be in multiple 
+states at the same time, and entanglement means the states can depend on each other. 
+In quantum computing systems, information is stored not using “bits”, but instead using 
+“qubits”.
+
+
+What is a qubit? 
+=================
+
+A qubit (pronounced “cue-bit”, a portmanteau of “quantum bit”) is the physical unit 
+of quantum information in quantum computing. It is the quantum version of a bit 
+(itself a portmanteau of “binary digit”), consisting of a two-state quantum mechanical 
+system that can (like a classical bit) exist in one state, `|0⟩`, or the other, `|1⟩`, 
+but unlike the classical bit counterpart, a qubit can also be in a quantum superposition 
+of both states.
+
+
+How do I access the OLCF quantum computing resources? 
+====================================================
+Applications for both Quantum Computing projects and quantum user accounts can be found 
+on the quantum accounts documentation :ref:`quantum-accounts` page.
+
+
+What happens after I apply for access to QCUP? 
+=============================================
+
+Applications are put through a merit review process, and you will be contacted 
+regarding the status of your application. See the user documentation page for more details.
+
+
+I formerly had access to quantum resources, but my backends/lattices/etc. have disappeared, what do I do? 
+=======================================================================================================
+
+If your account was established prior to July 5th, 2020, and was not through the OLCF directly, 
+your access to quantum resources has been removed, and you will need to re-apply to an OLCF project. 
+Also, if your access to your OLCF project or the project access itself has expired, you will also 
+need to reapply.
+
+
+I applied to a quantum computing resource via the vendor website, but don’t have access; what do I do? 
+====================================================================================================
```

Comment:
Need two more "=", it needs to match the heading length

Reply:
fixed